extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewManager.ts

```diff
@@ -166,13 +166,13 @@ export class PageViewManager {
                             if (!pageViewPerformance.isValid && !pageViewSent) {
                                 // If navigation timing gives invalid numbers, then go back to "override page view duration" mode.
                                 // That's the best value we can get that makes sense.
-                                customProperties["duration"] = customDuration;
+                                customProperties.duration = customDuration;
```

Comment:
This line is now converted automatically to `customProperties[strDuration] = customDuration;` because the `strDuration` is present in the `InternalConstants.ts`